src/index.ts

```diff
@@ -39,10 +41,14 @@ withAllStdIn((inputBuff: Buffer) => {
       thisFile.setContent(printFileDescriptorTSD(fileNameToDescriptor[fileName], exportMap));
       codeGenResponse.addFile(thisFile);
 
-      if (generateServices) {
+      if (generateGrpcWebServices) {
         generateGrpcWebService(outputFileName, fileNameToDescriptor[fileName], exportMap)
           .forEach(file => codeGenResponse.addFile(file));
       }
+      if (generateGrpcNodeServices) {
```

Comment:
This should be `else if` to reflect the fact that it's not possible to emit both `grpc-web` and `grpc-node` services at the same time.